src/components/design_system/breadcrumbs/__snapshots__/Breadcrumbs.test.tsx.snap

```diff
@@ -0,0 +1,227 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BreadcrumbsDesign renders empty for / 1`] = `
```

Comment:
Is the 1 here referring to snapshot v1?

Reply:
This file is generated by Jest. Jest will generate these .snap files for test files with tests that use `.toMatchSnapshot()` if they don't already exist. If the component code is changed so that it no longer matches the snapshot, the test will fail. To update the snapshot files to match the current version of the component (assuming changes to component are intended), you run `jest` with the `-u` or `--updateSnapshot` option. The way our project is set up in this PR branch, you can update all snapshot files with: 
```
npm run test:frontend -- --updateSnapshot
```

`BreadcrumbsDesign renders empty for /` is the name of the corresponding test in `Breadcrumbs.test.tsx` I'm not sure what the "1" refers to.  As long as it makes sense to Jest, I guess that's what matters. 😄 

See Jest docs for more on Snapshot testing: https://jestjs.io/docs/snapshot-testing

Reply:
@katconnors once Jest support is merged, would it make sense to document some of this in the README file or compass project wiki? Where do you think would be the best place?